cmd/make/make.go

```diff
@@ -0,0 +1,250 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"os/exec"
+	"path"
+	"path/filepath"
+	"runtime"
+	"strings"
+
+	"github.com/Masterminds/semver"
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/kolide/kit/fs"
+	"github.com/kolide/kit/logutil"
+	"github.com/pkg/errors"
+	"github.com/theupdateframework/notary/client"
+	"github.com/theupdateframework/notary/trustpinning"
+	"github.com/theupdateframework/notary/tuf/data"
+	"golang.org/x/sync/errgroup"
+)
+
+func main() {
+	var (
+		flTargets = flag.String("targets", "", "comma separated list of targets")
+		flDebug   = flag.Bool("debug", false, "use a debug logger")
+	)
+	flag.Parse()
+
+	logger := logutil.NewCLILogger(*flDebug)
+	b := builder{logger: logger}
+
+	targetSet := map[string]func() error{
+		"deps-go":       b.depsGo,
+		"install-tools": b.installTools,
+		"generate-tuf":  b.generateTUF,
+	}
+	targets := targetSet
```

Comment:
I did not want to just exec the provided targets because we might want to build some sort of dependency graph in the future. Imagine someone calls `generate,install-tools,deps-go`. these are not ordered. 

I agree that as it stands there's not much purpose to the intermediary set.

Reply:
First pass for dependency graphs can be pretty simpler -- functions can call what they depend one, with some map to hold exit codes, to keep from re-running a target. More complicated should mean we punt to an existing dependency resolver. (or bazel, etc) 

Up to you here. I'd simplify